gap/base/recognition.gi

```diff
@@ -555,7 +582,7 @@ InstallGlobalFunction( RecogniseGeneric,
         fi;
 
         if not IsReady(rifac) then
-            # the recognition of the image failed, also give up here:
+            # IsReady is not set, thus abort the whole computation.
             if InfoLevel(InfoRecog) = 1 and depth = 0 then Print("\n"); fi;
```

Comment:
Should we print here some information that the computation is aborted?

```suggestion
            if InfoLevel(InfoRecog) = 1 and depth = 0 then Print("\n"); fi;
            Info(InfoRecog, 1,
                 "ImageRecogNode of node <ri> could not be recognised,",
                 " IsReady(<ri>) is not set!");
```

Reply:
Good point. I'll add such an info message to the "unfinished case". And now that I think about it, we don't need to print anything in the "image" or "kernel" parts of RecogniseGeneric, since the warning message is printed in the "unfinished case" part already.